manifests/base/dex-server-role.yaml

```diff
@@ -1,14 +1,10 @@
----
 apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
 metadata:
   name: dex-server-role
 rules:
 - apiGroups:
   - ""
-  resourceNames:
```

Comment:
What is the rationale behind removing these restrictions?

Reply:
Good catch @merenbach . I think we still need names in namespace-install.yaml . Since Jesse is out , lets merge PR and restory name restriction in next PR

Reply:
The informer watch actually requires list access. So this change was necessary

Reply:
Thanks for letting us know, Jesse!